File: Electronics_Shop_17.Services/Database/PaymentMethod.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Reflection.Metadata.Ecma335;
using System.Text;
using System.Threading.Tasks;

namespace Electronics_Shop_17.Services.Database
{
    public class PaymentMethod : IHelperId
    {
        public int Id { get; set; }
        public string Code { get; set; }
        public string Type { get; set; }
        public string Provider { get; set; }
        
    }
}

File: angular17_web/src/app/models/payment-method/payment.method.ts
export class PaymentMethod {
  id!: number;
  code!: string;
  type!: string;
  provider!: string;

  constructor(data?: any) {
    if (!data) return;

    this.id = data.id;
    this.code = data.code;
    this.type = data.type;
    this.provider = data.provider;
  }
}
